spotify-api.el

```diff
@@ -228,6 +228,45 @@ depending on the `type' argument."
            name
            (if is-public "true" "false"))))
 
+(defun spotify-api-album-get-tracks (album-id)
+  "Get list of track ids for given album."
+  (mapcar 'spotify-get-item-id (spotify-get-items (spotify-api-call
+						   "GET"
+						   (format "/albums/%s/tracks"
+							   (url-hexify-string album-id))))))
+
+(defun spotify-api-playlist-add-track (user-id playlist-id track-id)
+  "Add single track to playlist"
+  (spotify-api-playlist-add-tracks user-id playlist-id (list track-id)))
+
+
+(defun spotify-format-id (type id)
+  "Wrap raw id to type if necessary"
+  (if (string-match-p "spotify" id)
+      (format "\"%s\"" id)
+    (format "\"spotify:%s:%s\"" type id)))
+
+
+(defun spotify-api-playlist-add-tracks (user-id playlist-id track-ids)
+  "Add tracks in list track-ids in playlist"
+  (let ((tracks (format "%s" (mapconcat (lambda (x) (spotify-format-id "track" x)) track-ids ","))))
+      (progn
+    (message "Adding: %s" tracks)
```

Comment:
I'd remove the info message from this function and put it inside the `spotify-track-add`, in `spotify-track-search.el`.

Reply:
Yeah that was kinda confusing